python/ray/dataframe/test/test_dataframe.py

```diff
@@ -1673,11 +1676,8 @@ def test___iter__():
         ray_df.__iter__()
 
 
-def test___contains__():
-    ray_df = create_test_dataframe()
-
-    with pytest.raises(NotImplementedError):
-        ray_df.__contains__(None)
+def test___contains__(ray_df, pandas_df, value):
+    assert(ray_df.__contains__(value) == pandas_df.__contains__(value))
```

Comment:
Try looping over values for this test (see `test_apply_map()`. Also, `__contains__()` is looking for columns, not values within the dataframe. In your tests it will always evaluate to `False` in both cases.

Reply:
Resolved by calling __contains__ for each key